packages/network-clients/codegen.ts

```diff
@@ -0,0 +1,55 @@
+// Copyright 2020-2022 SubQuery Pte Ltd authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import { CodegenConfig } from '@graphql-codegen/cli'
```

Comment:
Can we move the types and type generation script to `network-query`, and export the types.
These types are generated from `*.gql`, could be put them in the same package, in that case the query and types can maintain together to make the sure the interface is consistent.